pkg/prometheus/rules_test.go

```diff
@@ -129,7 +129,7 @@ func shouldReturnAtLeastOneConfigMap(t *testing.T) {
 		t.Fatalf("expected no error but got: %v", err.Error())
 	}
 
-	if len(configMaps) != 1 {
+	if len(configMaps) != 3 {
 		t.Fatalf("expected one ConfigMaps but got %v", len(configMaps))
```

Comment:
forgot to update the error message
```suggestion
		t.Fatalf("expected three ConfigMaps but got %v", len(configMaps))
```